components/BudgetItemsList.js

```diff
@@ -302,7 +302,13 @@ const BudgetItem = ({ item, isInverted, isCompact, canDownloadInvoice, intl }) =
   return (
     <React.Fragment>
       <ItemContainer>
-        <Flex p={[3, 24]} flexWrap="wrap" alignItems={['center', 'flex-start']}>
+        <Flex
+          data-type="transactions"
```

Comment:
The reason why we use `data-cy` is that it gets stripped out of the attributes during the build so that it's not included in production code. We should stick with `data-cy` as much as possible and remove this `data-type` if we can avoid it.